bigbluebutton-html5/imports/ui/components/video-provider/video-list/video-list-item/component.jsx

```diff
@@ -102,7 +102,7 @@ class VideoListItem extends Component {
     const { user } = this.props;
 
     const availableActions = this.getAvailableActions();
-    const isVideoMenuDisabled = Meteor.settings.public.kurento.disableVideoMenu || false;
+    const isVideoMenuDisabled = Meteor.settings.public.kurento.enableHideVideoMenu || false;
```

Comment:
It would be 'enabled...' here too - and likely the logic will have to be flipped